### PR TITLE
Added feedback buttons to emails

### PR DESCRIPTION
**Pull request by lenabaidakova**
closes TryGhost/Team#2075

### CODE REVIEW

**Comment by lenabaidakova** [comment]
Need to improve for Outlook https://github.com/TryGhost/Team/issues/2167

**Comment by moreofmorris** [comment]
Hey @lenabaidakova, I just tested in Gmail app and this is what shown for me

CleanShot 2022-10-26 at 10 40 48@2x

I'm not sure if this helps simplify this but the more I think about it, the most we should probably just use black for the icon and text? That would massively simplify this, is that right?

**Review by moreofmorris — APPROVED** [review]

File: ghost/core/core/server/services/mega/feedback-buttons.js [hunk 1]
```diff
@@ -18,15 +18,27 @@ const generateLinks = (postId, uuid, html) => {
...
 
 const getTemplate = (accentColor) => {
-    const likeButtonHtml = getButtonHtml(templateStrings.like, 'More like this', accentColor);
-    const dislikeButtonHtml = getButtonHtml(templateStrings.dislike, 'Less like this', accentColor);
+    const likeButtonHtml = getButtonHtml(
+        templateStrings.like,
+        'More like this',
+        accentColor,
+        'like-icon',
+        'https://static.ghost.org/v4.0.0/images/thumbs-up.png'
```
Comment (SimonBackx) [inline ghost/core/core/server/services/mega/feedback-buttons.js:33]: I think at this point it would be better to upload them at /v5.0.0, no?

**Review by SimonBackx — APPROVED** [review]
Nice change! Only left a small comment

**Pull request merged**